### ISSUE TITLE
Add version number to help output

### ISSUE THREAD

**Issue by jtladner** [LadnerLab/PepSIRF#45]
Please add version number information to the message printed to the screen when PepSIRF is invoked with "-h". This should include invocations that do out specify a module (e.g., "pepsirf -h") and those that do specify a module (e.g., "pepsirf demux -h"). 

Currently, version number is only included in the output of the script generated during an actual analysis. However, there is no way to check the version prior to running an analysis.

**Comment by ZwFink** [comment]
Do you have the latest version of PepSIRF pulled? 
Once you are able to build, you will see:
```
finktop :: Python/PepSIRF/build ‹master*› » pepsirf demux -h                                                                                                                              1 ↵
PepSIRF (v1.2.1): Peptide-based Serological Immune Response Framework demultiplexing module.
```

```
finktop :: Python/PepSIRF/build ‹master*› » pepsirf -h
PepSIRF (v1.2.1): Peptide-based Serological Immune Response Framework

```

**Comment by jtladner** [comment]
You're right. This is already fixed in 1.2.1
